src/applications/bmqtool/m_bmqtool_storageinspector.cpp

```diff
@@ -474,7 +474,7 @@ void StorageInspector::processCommand(
             fields.push_back("SyncPoint DataFileOffset (DWORDS)");
             fields.push_back("SyncPoint QlistFileOffset (WORDS)");
 
-            bmqu::AlignedPrinter printer(BALL_LOG_OUTPUT_STREAM, &fields);
+            bmqu::AlignedPrinter printer(BALL_LOG_OUTPUT_STREAM, fields);
```

Comment:
Also need to change type above, and do the same in other places in this file:
`bsl::vector<const char*> fields` -> `bsl::vector<bsl::string> fields`